### DISCUSSION TITLE
Unable to connect OAP to MySql

### DISCUSSION THREAD

**Discussion by SooryaMuthuraj** [apache/skywalking#12183] [General]
Dear All,

So I have been trying to connect the OAP to mySql for storing APM data but unable to connect. The error has been attached below and please us to solve the issue.

Thanks. 

![image](https://github.com/apache/skywalking/assets/154515976/e2adb112-3ef4-4b3c-b36b-b98aa9f7b0e1)

**Comment by wu-sheng** [comment]
https://skywalking.apache.org/docs/main/latest/en/setup/backend/storages/mysql/


Please read the docs and pay attention on notice. We don't have MySQL driver in the binary by following ASF policy.

**Reply by SooryaMuthuraj** [reply]
But in Centos 7 our installation was successful

**Reply by wu-sheng** [reply]
I am highly doubt. You could read the logs, it says no driver, and truely, we don't distribute binary with MySQL driver.
